### ISSUE TITLE
Community Feedback Mechanism: What HTML5 Features Do They Want to See More Of?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#181]
```
Would be very useful to have a poll on the site so readers can vote for the 
next set of material.  It would be a good indicator of what HTML5 features need 
more explanation, examples, or articles.

This can coincide with our current plans to create a page-per-feature.  We can 
have a simple feedback request like "Want to see more about this feature? 
Yes/No"
```

Original issue reported on code.google.com by `sethladd@google.com` on 23 Aug 2010 at 10:22

**Comment by GoogleCodeExporter** [comment]
```
Lets use Google Moderator for this :)
```

Original comment by `ericbide...@html5rocks.com` on 7 Sep 2010 at 8:44
- Added labels: **Type-Enhancement**, **Milestone-X**
- Removed labels: **Type-Task**

**Comment by GoogleCodeExporter** [comment]
```
Good call!
```

Original comment by `sethladd@google.com` on 7 Sep 2010 at 9:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 8 Sep 2010 at 12:14
- Changed state: **Started**
- Added labels: **Milestone-3**
- Removed labels: **Milestone-X**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 58036b5bae.
```

Original comment by `ericbide...@html5rocks.com` on 25 Sep 2010 at 6:15
- Changed state: **Fixed**
